src/ProjectTemplates/test/MvcTemplateTest.cs

```diff
@@ -234,68 +227,44 @@ public async Task MvcTemplate_IndividualAuth(bool useLocalDB)
         }
     }
 
-    [ConditionalFact(Skip = "https://github.com/dotnet/aspnetcore/issues/25103")]
+    [ConditionalFact]
+    [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)] // Running these requires the rid-specific runtime pack to be available which is not consistent in all our platform builds.
     [SkipOnHelix("cert failure", Queues = "All.OSX")]
     public async Task MvcTemplate_SingleFileExe()
     {
         // This test verifies publishing an MVC app as a single file exe works. We'll limit testing
         // this to a few operating systems to make our lives easier.
-        string runtimeIdentifer;
-        if (OperatingSystem.IsWindows())
-        {
-            runtimeIdentifer = "win-x64";
-        }
-        else if (OperatingSystem.IsLinux())
-        {
-            runtimeIdentifer = "linux-x64";
-        }
-        else
-        {
-            return;
-        }
-
+        var runtimeIdentifer = "win-x64";
         var project = await ProjectFactory.GetOrCreateProject("mvcsinglefileexe", Output);
         project.RuntimeIdentifier = runtimeIdentifer;
 
-        var createResult = await project.RunDotNetNewAsync("mvc", auth: "Individual", useLocalDB: true);
+        var createResult = await project.RunDotNetNewAsync("mvc");
         Assert.True(0 == createResult.ExitCode, ErrorMessages.GetFailedProcessMessage("create/restore", project, createResult));
 
         var publishResult = await project.RunDotNetPublishAsync(additionalArgs: $"/p:PublishSingleFile=true -r {runtimeIdentifer}", noRestore: false);
         Assert.True(0 == publishResult.ExitCode, ErrorMessages.GetFailedProcessMessage("publish", project, publishResult));
 
-        var pages = new[]
+        var menuLinks = new[]
         {
-                new Page
-                {
-                    // Verify a view from the app works
-                    Url = PageUrls.HomeUrl,
-                    Links = new []
-                    {
-                        PageUrls.HomeUrl,
-                        PageUrls.RegisterUrl,
-                        PageUrls.LoginUrl,
-                        PageUrls.HomeUrl,
-                        PageUrls.PrivacyUrl,
-                        PageUrls.DocsUrl,
-                        PageUrls.PrivacyUrl
-                    }
-                },
-                new Page
-                {
-                    // Verify a view from a RCL (in this case IdentityUI) works
-                    Url = PageUrls.RegisterUrl,
-                    Links = new []
-                    {
-                        PageUrls.HomeUrl,
-                        PageUrls.RegisterUrl,
-                        PageUrls.LoginUrl,
-                        PageUrls.HomeUrl,
-                        PageUrls.PrivacyUrl,
-                        PageUrls.ExternalArticle,
-                        PageUrls.PrivacyUrl
-                    }
-                },
-            };
+            PageUrls.HomeUrl,
```

Comment:
```suggestion
            PageUrls.HomeUrl,
            PageUrls.HomeUrl,            
```

Apparently this is intentional because there are two links to the Home page on the landing page.